rebuild_test.py

```diff
@@ -140,7 +140,9 @@ def resumable_rebuild_test(self):
         """
         self.ignore_log_patterns = list(self.ignore_log_patterns) + [r'Error while rebuilding node',
                                                                      r'Streaming error occurred on session with peer 127.0.0.3',
-                                                                     r'Remote peer 127.0.0.3 failed stream session']
+                                                                     r'Remote peer 127.0.0.3 failed stream session',
+                                                                     r'Streaming error occurred on session with peer 127.0.0.3:7000',
```

Comment:
how will this work if we use a single (network) interface? or configurable ports? is it best to get the address of node3 and plug that into the patterns, or is that opening up a large can of worms?